content/_data/changelogs/weekly.yml

```diff
@@ -87,7 +87,82 @@
   date: 2017-03-11
   changes:
     - type: rfe
-      message: Expose the noun for an item when behaving as a task via <tt>AlternativeUiTextProvider</tt>.
+      message: >
+        Allow searching by build parameter values in the <code>Build History</code> widget.
```

Comment:
@oleg-nenashev Both are inappropriate IMO. *em*, **strong** if you need to emphasize something. But neither this nor the versions below qualify.